pkg/kubefedctl/orphaning/status.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package orphaning
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/klog"
+
+	ctlutil "sigs.k8s.io/kubefed/pkg/controller/util"
+	"sigs.k8s.io/kubefed/pkg/kubefedctl/util"
+)
+
+const (
+	Enabled  = "Enabled"
+	Disabled = "Disabled"
+)
+
+var (
+	orphaning_status_long = `
+		Checks the status of "orphaning enable" ('kubefed.k8s.io/orphan: true')
+		annotation on a federated resource. 
+		Returns "Enabled" or "Disabled"
```

Comment:
nit: Please move the content on this line to the previous line.

Reply:
Hope I understand you, done

Reply:
I meant add ensuring `Returns "Enabled"...` wasn't separated from the previous sentence with a newline, but this is fine too.